metrics/src/metrics.rs

```diff
@@ -245,6 +245,7 @@ impl MetricsAgent {
 
         let extra = influxdb::Point::new("metrics")
             .add_timestamp(timing::timestamp() as i64)
+            .add_tag("host_id", influxdb::Value::String(HOST_ID.to_string()))
```

Comment:
Sending `host_id` as both a tag and field is a little yucky for bandwidth.  Can we just send `host_id` as a tag?